src/core/src/bootstrap/EnvLayer.py

```diff
@@ -65,47 +47,42 @@ def is_distro_azure_linux(distro_name):
         return any(x in distro_name for x in Constants.AZURE_LINUX)
 
     def get_package_manager(self):
+        # type: () -> str
         """ Detects package manager type """
-        ret = None
+        if platform.system() == 'Windows':
+            return Constants.APT
```

Comment:
Note to look for in this PR: All callers must have earlier been implemented to expect None value, if a pkgmgr is not found. That should change to empty string